### PR TITLE
Nytimes website clone

### PR DESCRIPTION
**Pull request by ershadul1**
We have created newyork times article clone page in a branch named 
nytimes-website-clone. 

- We applied floats

- We applied flexbox

- We applied grid 

- We applied position

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,428 @@
...
+        </div>
```
Comment (peterrobert) [inline index.html]: This logo should be a link

**Review by peterrobert — CHANGES_REQUESTED** [review]
### CHANGES REQUIRED :recycle: 
**GREAT PROJECT** :+1:  You have done such an impressive job :clap:  :100: 
However, there are a few things that have prevented me from proving this project kindly improve on the following :smile: 

- [x] Your readme file can be improved. I saw that you left a lot of blank areas in your readme file. Kindly fill in these areas. Most of them can be filled.  Let's make the readme file more descriptive :smile: 

- [x] Your pull request should state using bullet points what you have worked on in this project:smile: 
Kindly do that.

- [x] This section on your webpage does not look like the original kindly fix that. 
yours :arrow_down: 
more-space
clone :arrow_down: 
more-space-original

- [x] Also this part needs improvement 
yours :arrow_down: 
subscribenow
original :arrow_down: 
subscribenoworig

Kindly improve on those and submit your work for another review :+1: 

If you have any question kindly contact me via slack on @peterrobert

### HAPPY CODING

File: index.html (hunk 1 above)
Comment (ershadul1) [reply index.html]: We have fixed all the issues.

**Comment by addod19** [comment]
## Pull Request under review by Daniel Larbi Addo (TSE)

**Review by addod19 — APPROVED** [review]
## Status: Approved :trophy: :balloon: 

Hi @ershadul1 and @Haywhizzz, great job fixing all the comments made by the previous TSE :+1: Congratulations on this great job. :100: 
[OPTIONAL]
- Kindly center this content, it is not centered currently
cntr

Go ahead and merge to master :slightly_smiling_face: 
_Happy coding_ :man_technologist: 
[Daniel Larbi Addo](https://www.linkedin.com/in/daniel-larbi-addo/)

**Pull request merged**